handlers/feedback.go

```diff
@@ -90,6 +92,14 @@ func getFeedback(w http.ResponseWriter, req *http.Request, url, errorType, descr
 	p.Type = "feedback"
 	p.Metadata.Title = "Feedback"
 	p.Metadata.Description = url
+	ctx := context.Background()
+	mappedNavContent, _ := cacheService.GetMappedNavigationContent(ctx, lang)
+	fmt.Println("test")
+	//if err == nil {
+	fmt.Println(mappedNavContent)
+
+	p.NavigationContent = mappedNavContent
+	//}
 
```

Comment:
Think there's some comments here that need to be removed as well as the `fmt.Println`? Guessing it might be out of scope for this ticket but imagine there should be in a mapper file... Shouldn't there be a check for `nil`?